editor/js/Sidebar.Scene.js

```diff
@@ -181,7 +181,23 @@ function SidebarScene( editor ) {
 	backgroundEquirectangularTexture.setDisplay( 'none' );
 	backgroundRow.add( backgroundEquirectangularTexture );
 
+	const backgroundColorSpaceRow = new UIRow();
+	backgroundColorSpaceRow.setDisplay( 'none' );
+	backgroundColorSpaceRow.setMarginLeft( '120px' );
+
+	const backgroundColorSpace = new UISelect().setOptions( {
+
+		'': 'No Color Space',
+		'srgb-linear': 'sRGB (linear)',
+		'srgb': 'sRGB',
```

Comment:
Can use the constants if you'd prefer! Note that the values of the constants are copied directly from CSS (https://web.dev/blog/color-spaces-and-functions#access_color_spaces_from_css), so those could be OK as labels too.

```suggestion
		[THREE.LinearSRGBColorSpace]: 'sRGB (linear)',
		[THREE.SRGBColorSpace]: 'sRGB',
```